polkadot/xcm/xcm-executor/src/traits/weight.rs

```diff
@@ -31,6 +31,7 @@ pub trait WeightBounds<RuntimeCall> {
 
 /// A means of getting approximate weight consumption for a given destination message executor and a
 /// message.
+#[allow(dead_code)]
```

Comment:
Why is this dead code? Is the trait not reachable form the outside? Then we should remove it.